docs/settings/infrastructure-ui-settings.asciidoc

```diff
@@ -0,0 +1,39 @@
+[role="xpack"]
+[[infrastructure-ui-settings-kb]]
+=== Infrastructure UI Settings in Kibana
+++++
+<titleabbrev>Infrastructure UI Settings</titleabbrev>
+++++
+
+You do not need to configure any settings to use Infrastructure UI. It is enabled by default.
+
+[float]
+[[general-infra-ui-settings-kb]]
+==== General Infrastructure UI Settings
+
+xpack.infra.enabled:: Set to `false` to disabled the Logs and Infrastructure UI plugin {kib}. Defaults to
+`true`.
+
+xpack.infra.sources.default.logAlias:: Index pattern for matching indices that contain log data. Defaults to
+`filebeat-*`.
+
+xpack.infra.sources.default.metricAlias:: Index pattern for matching indices that contain Metricbeat data. Defaults to
+`metricbeat-*`.
+
+xpack.infra.sources.default.fields.timestamp:: Timestamp used to for sorting log entries. Defaults to
+`@timestamp`.
+
+xpack.infra.sources.default.fields.message:: Fields used for the message for Logs UI. Defaults to
```

Comment:
Would it be correct to say, "Fields used to pass log messages to the Logs UI"?

I think by "message" you mean each log message, but I'm not sure. I don't have the UI open right now to investigate, but just wanted to note that this is a little ambiguous and the wording seems off.